### PR TITLE
#162163249 Reading stats of all users.

### PR DESCRIPTION
**Pull request by KicaRonaldOkello**
### What does this PR do?
This PR ensures that read and views count per article is recorded when the article is opened. The statistics( reads and views) of the articles are displayed to the author of the article.
#### Description of Task to be completed?

- Record both read time and view time of an article.

- Display the reads and views to the author of the article.

#### How should this be manually tested?

- Create an article.

- Enter go to your profile page and click the button `Your read statistics`

- The article will have 0 reads and 0 views.

- Open the article and read it.

- Go back and check on your reading statistics, the views and/or reads values will have increased depending on how much time you spent on the article.

#### What are the relevant pivotal tracker stories?
[Delivers #162163249]
#### Screenshots:
image
Screenshot of newly created article
image

Click the lower button
image
View read stats of the article.
image
Read stats once the article has been viewed

### CODE REVIEW

**Review by Kasulejoseph — APPROVED** [review]
it looks good

**Review by bibangamba — CHANGES_REQUESTED** [review]
Hi @KicaRonaldOkello, this is great stuff. I like the UI for a single reading stat. However, there seems to be some bugs; 
1. some CSS might have changed and affected your design (specifically the spacing). Each stat is on a page of its own, which I know you'd designed to look closer to a list of stats from previous iterations. please fix that.
2. there is another bug related to the articles shown. It seems to show stats for all articles. not just my own.
Other than these 2 issues, the functionality looks good.

**Comment by Nta1e** [comment]
Stats are too spaced. Just improve on the styling.
Otherwise, this is good work!

**Review by Nta1e — CHANGES_REQUESTED** [review]
Stats are too spaced. Just improve on the styling.
Otherwise, this is good work!

File: src/actions/actionTypes.js [hunk 1]
```diff
@@ -31,11 +31,13 @@ const actionTypes = {
...
-  EDITCOMMENT: 'EDITCOMMENT',
-  COMMENTDELETE: 'COMMENTDELETE',
+  COMMENTSUCCESS: "COMMENTSUCCESS",
+  COMMENTERROR: "COMMENTERROR",
+  REPLYSUCCESS: "REPLYSUCCESS",
+  GETCOMMENT: "GETCOMMENT",
+  EDITCOMMENT: "EDITCOMMENT",
+  COMMENTDELETE: "COMMENTDELETE",
+  READ_TIME: "READ_TIME",
+  MY_ARTICLES: "MY_ARTICLES"
```
Comment (JaymesKat) [inline src/actions/actionTypes.js]: `MY_ARTICLES` is not a proper word for an action. It should include a verb implying something that is done @KicaRonaldOkello

File: src/actions/articleActions/ArticleActions.js [hunk 2]
```diff
@@ -110,11 +110,25 @@ const getMoreArticles = (page, searchParam = "") => dispatch => {
     .catch(err => err);
 };
 
+const articleReadCount = (slug, time) => dispatch => {
```
Comment (JaymesKat) [inline src/actions/articleActions/ArticleActions.js]: @KicaRonaldOkello please rename the action to something like `getArticleReadCount` to imply the action being performed

**Review by JaymesKat — CHANGES_REQUESTED** [review]

**Review by bibangamba — APPROVED** [review]
Great stuff!!

**Review by Nta1e — APPROVED** [review]
@KicaRonaldOkello, Good work👏🏽

**Review by JaymesKat — APPROVED** [review]

**Pull request merged**